lib/internal/Magento/Framework/Interception/Test/Unit/Code/Generator/_files/Interceptor.txt

```diff
@@ -50,4 +50,17 @@ class Interceptor extends \Magento\Framework\Interception\Code\Generator\Sample
             return $this->___callPlugins('getReference', func_get_args(), $pluginInfo);
         }
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function variadic(... $values)
```

Comment:
It would be nice to extend this case to support multiple arguments before variadic, and type-hinted variadic.

Reply:
Also, do you think that space should be used after `...`? As far as I can see it is usually omitted.

Reply:
Uniformity with [zend-code ParameterGenerator](https://github.com/zendframework/zend-code/blob/develop/src/Generator/ParameterGenerator.php#L304)